lib/Cmts.ml

```diff
@@ -628,16 +628,19 @@ let drop_inside t loc =
   in
   clear t.cmts_before ; clear t.cmts_within ; clear t.cmts_after
 
+let pop_if_debug t loc =
+  if Conf.debug && t.remove then Hashtbl.remove t.remaining loc
```

Comment:
Is it really defined twice?

Reply:
yes, I moved it but forgot to remove the original one. Thanks!